File: C++/Program/Civitas.cpp
#include "Human.cpp"

using namespace std;

class Civitas : public Human
{
private:
    string univ;
    string email;

public:
    Civitas()
    {
        this->univ = "";
        this->email = "";
    }

    string GetUniv() const
    {
        return this->univ;
    }

    void SetUniv(string univ)
    {
        this->univ = univ;
    }

    string GetEmail() const
    {
        return email;
    }

    void SetEmail(string email)
    {
        this->email = email;
    }

    ~Civitas()
    {
    }
};

File: PHP/Civitas.php
<?php

class Civitas extends Human
{
    // atribut privat
    private $univ, $email;

    // constructor
    public function __construct()
    {
        $this->univ = "";
        $this->email = "";
    }

    /**
     * Get the value of email
     */
    public function getEmail()
    {
        return $this->email;
    }

    /**
     * Set the value of email
     *
     */
    public function setEmail($email)
    {
        $this->email = $email;
    }

    /**
     * Get the value of univ
     */
    public function getUniv()
    {
        return $this->univ;
    }

    /**
     * Set the value of univ
     *
     */
    public function setUniv($univ)
    {
        $this->univ = $univ;
    }

    // print civitas
    function printCivitas()
    {
        echo "Data Civitas :" . $this->getUniv() . " --- " . $this->getEmail() . "<br/>";
    }

    // destruct
    public function __destruct()
    {
    }
}
